lib/cc/engine/analyzable_files.rb

```diff
@@ -1,39 +1,24 @@
 module CC
   module Engine
     class AnalyzableFiles
-      def initialize(directory, config)
-        @directory = directory
+      def initialize(config)
         @config = config
       end
 
       def all
-        @results ||= if @config["include_paths"]
-          build_files_with_inclusions(@config["include_paths"])
+        @results ||= if include_paths = @config["include_paths"]
+          filter_files(include_paths)
         else
-          build_files_with_exclusions(@config["exclude_paths"] || [])
+          raise "PLACEHOLDER: We don't support exclude_paths any longer!"
         end
       end
 
       private
 
-      def fetch_files(paths)
-        paths.map do |path|
-          if path =~ %r{/$}
-            Dir.glob("#{path}/**/*.coffee")
-          else
-            path if path =~ /\.coffee$/
-          end
-        end.flatten.compact
-      end
-
-      def build_files_with_inclusions(inclusions)
-        fetch_files(inclusions)
-      end
-
-      def build_files_with_exclusions(exclusions)
-        files = Dir.glob("#{@directory}/**/*.coffee")
-        excluded_files = fetch_files(exclusions)
-        files.reject { |f| exclusions.include?(f) }
+      def filter_files(files)
+        files.select do |file|
+          file.end_with?("/") || file.end_with?(".coffee")
```

Comment:
Can you check if it's a directory instead of the ending? 

```
irb(main):002:0> File.directory?("lib")
=> true
irb(main):003:0> File.directory?("lib/")
=> true
irb(main):004:0> 
```